### PR TITLE
docs(Bram): A sad farewell

### PR DESCRIPTION
**Pull request by Computer2340**
I thought maybe an appropriate way to handle
https://github.com/neovim/neovim/issues/24579 would be with putting a doc about Bram directly in the NeoVim manual. I'm sure other people have more that they could say, and I'm also totally willing to let this PR be closed because it's off topic for the rest of the docs but I just sort of felt like something should be distributed with every copy of NeoVim that said a little bit about how at least some of us feel right now.

### CODE REVIEW

**Comment by justinmk** [comment]
Not bad. Should live in `:help intro` somewhere.

File: runtime/doc/intro.txt [hunk 1]
```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *bram*
```
Comment (clason) [inline runtime/doc/intro.txt:197]:
```suggestion
Bram Moolenaar                                    *Bram* *brammool*
```

I like it; it's respectful without being cloying.

File: runtime/doc/intro.txt [hunk 2]
```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
...
+
+NeoVim is a text editor built as a continuation of Vim, Vi Improved. Vim was
+orginally developed by Bram Moolenaar.  Searching his name within the source
+code of NeoVim will reveal just how much of his work still remains in
+NeoVim.  On August 3, 2023, he passed away at the age of 62.  If Vim or NeoVim
```
Comment (clason) [inline runtime/doc/intro.txt:202]:
```suggestion
Nvim is a text editor built as a continuation of Vim, Vi Improved. Vim was
orginally developed by Bram Moolenaar.  Searching his name within the source
code of Nvim will reveal just how much of his work still remains in
Nvim.  On August 3, 2023, he passed away at the age of 62.  If Vim or Nvim
```
The text editor is called "Nvim" (launched through the command `nvim`). "Neovim" is the whole project.

File: runtime/doc/intro.txt [hunk 3]
```diff
@@ -71,12 +71,12 @@ If Nvim crashes, try to get a backtrace.  See |debug.txt|.
 Sponsor Vim/Nvim development				*sponsor* *register*
 
 Fixing bugs and adding new features takes a lot of time and effort.  To show
-your appreciation for the work and motivate Bram and others to continue
+your appreciation for the work and motivate Vim developers to continue
```
Comment (clason) [inline runtime/doc/intro.txt:74]: That's the only thing I'm not 100% sure about; I'd prefer not to write anything on somebody else's behalf -- certainly not the Vim developers, who haven't made any official statements yet. Maybe just combine these two paragraphs and end this sentence after "show your appreciation for the work of Bram"?

(Or just not touch this section at all and wait what Vim will write here, once they have sorted everything out. There's no reason to rush this.)

**Review by clason — CHANGES_REQUESTED** [review]

File: runtime/doc/intro.txt (hunk 3 above)
Comment (Computer2340) [reply runtime/doc/intro.txt:74]: I went back and forth on changes to this section. I'll leave it untouched for now, and the new section can be the extent of the changes

Comment (Computer2340) [reply runtime/doc/intro.txt]: Removed

File: runtime/doc/intro.txt [hunk 4]
```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *brammool*
+
+Nvim is a text editor built as a continuation of Vim, Vi Improved.  Vim was
+orginally developed by Bram Moolenaar.  Searching his name within the source
+code of Nvim will reveal just how much of his work still remains in
+nvim.  On August 3, 2023, he passed away at the age of 62.  If Vim or Nvim
```
Comment (zeertzjq) [inline runtime/doc/intro.txt:202]:
```suggestion
Nvim.  On August 3, 2023, he passed away at the age of 62.  If Vim or Nvim
```

File: runtime/doc/intro.txt [hunk 5]
```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *brammool*
+
+Nvim is a text editor built as a continuation of Vim, Vi Improved.  Vim was
+orginally developed by Bram Moolenaar.  Searching his name within the source
```
Comment (gpanders) [inline runtime/doc/intro.txt:200]:
```suggestion
originally developed by Bram Moolenaar.  Searching his name within the source
```

**Review by justinmk — APPROVED** [review]

**Comment by Computer2340** [comment]
@justinmk , do I need  to squash my commits so that the github generated "change accepted" commits aren't upsetting the linter?

**Comment by clason** [comment]
We can (and will) squash when merging.

File: runtime/doc/intro.txt [hunk 6]
```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *brammool*
+
+Nvim is a text editor built as a continuation of Vim, Vi Improved.  Vim was
```
Comment (clason) [inline runtime/doc/intro.txt:199]:
```suggestion
Nvim is a text editor built as a continuation of Vim ("Vi IMproved").  Vim was
```

**Review by clason — APPROVED** [review]

File: runtime/doc/intro.txt (hunk 6 above)
Comment (zeertzjq) [inline runtime/doc/intro.txt:199]: I also don't think "continuation" is the right word here.

Comment (clason) [reply runtime/doc/intro.txt:199]: "fork" is how we usually describe it

**Comment by mifopen** [comment]
@justinmk Just wanted to point out that https://github.com/neovim/neovim/wiki/Introduction has few words about Bram in present tense. Maybe it should be adjusted as well?

**Comment by justinmk** [comment]
Thanks. We are migrating the wiki almost entirely to `:help`, so we'll eventually get to that page.

**Pull request merged**